rollingpin/deploy.py

```diff
@@ -118,21 +119,50 @@ def signal_handler(sig, _):
                 yield self.event_bus.trigger("build.begin")
 
                 try:
-                    build_command = ["build"] + components
```

Comment:
Previously, sync + build were in the same step so it was all smooshed together. For some repos (e.g. metron) it would sync properly then run reddit's build process for no reason. Now it'll just sync and skip straight to deploying.